### PR TITLE
Make /var/log/audit permissions rules actually check log_file in auditd.conf

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- Add shared check auditd_conf_log_file_not_set
- Altered bash and oval of file_permissions_var_log_audit, file_ownership_var_log_audit_stig, file_group_ownership_var_log_audit and directory_permissions_var_log_audit to consider log_file parameter, if not fallback to default /var/log/audit
- Added those rules to Ubuntu 20.04 STIG profile
- Added some tests as well

### CODE REVIEW

**Review by jan-cerny — COMMENTED** [review]
Thanks for this PR

File: linux_os/guide/system/auditing/auditd_configure_rules/file_ownership_var_log_audit_stig/oval/shared.xml [hunk 1]
```diff
@@ -1,23 +1,36 @@
 <def-group>
   <definition class="compliance" id="file_ownership_var_log_audit_stig" version="1">
-    {{{ oval_metadata("Checks that all /var/log/audit files are owned by the root user.") }}}
-    <criteria comment="files are root owned">
-      <criterion test_ref="test_user_ownership_var_log_audit_files" />
+    {{{ oval_metadata("Checks that all audit log files are owned by the root user.") }}}
+    <criteria operator="OR">
+      <criteria operator="AND" comment="log_file set">
+        <extend_definition comment="log_file not set in auditd.conf" definition_ref="auditd_conf_log_file_not_set" negate="true" />
+        <criterion comment="audit log files are root owned" test_ref="test_user_ownership_audit_log_files" />
```
[the remark this answers is not in the document]
Comment (dodys) [reply linux_os/guide/system/auditing/auditd_configure_rules/file_ownership_var_log_audit_stig/oval/shared.xml:7]: Added a few comments addressing your comments. Thanks!

**Review by jan-cerny — APPROVED** [review]
Thanks for your PR. Great job!

**Pull request merged**